src/content/docs/en/reference/cli-reference.mdx

```diff
@@ -373,3 +373,125 @@ Automatically opens the app in the browser on server start.
 ### `--help`
 
 Prints the help message and exits.
+
+## Advanced APIs (Experimental)
+
+If you need more control when running Astro, the `"astro"` package also exports APIs to programmatically run the CLI commands. These APIs are experimental and their API signature may change in any version.
+
+### `AstroInlineConfig`
+
+The `AstroInlineConfig` type is used by all of the command APIs below. It extends from the user [Astro config](/en/reference/configuration-reference/) type:
+
+```ts
+interface AstroInlineConfig extends AstroUserConfig {
+	configFile?: string | false;
+	mode?: "development" | "production";
+	logLevel?: "debug" | "info" | "warn" | "error" | "silent";
+}
+```
+
+#### `configFile`
+
+<p>
+
+**Type:** `string | false`<br />
+**Default:** `undefined`
+</p>
+
+A custom path to the Astro config file. If relative, it'll resolve based on the current working directory.
+Set to `false` to disable loading any config files.
+
+If this value is undefined or unset, Astro will search for an `astro.config.(js,mjs,ts)` file relative to
+the `root` and load the config file if found.
```

Comment:
```suggestion
A custom path to the Astro config file. 

If this value is undefined (default) or unset, Astro will search for an `astro.config.(js,mjs,ts)` file relative to
the `root` and load the config file if found.

If a relative path is set, it will resolve based on the current working directory.

Set to `false` to disable loading any config files.
```
I love how clear this all is! So well written! I just thought maybe separating out the one-liner definition makes it stand out more, and swapped the order around to mention the default behaviour first.